games/s.yaml

```diff
@@ -401,6 +401,22 @@
   type: remake
   updated: 2018-06-15
 
+- name: 'Trust & Betrayal: The Legacy of Siboot'
```

Comment:
this looks like the wrong file

Reply:
I added at originals too, but i don't know if it's ok

Reply:
the files under games/ are for clones
the originals go under originals/
the two types of files have different schemas, so this stuff doesn't belong here

Reply:
I checked this syntax with https://osgameclones.com/add_original.html as you recommend and pushed,
would you tell me please if it's correct?

Reply:
So, you are saying instead of adding this game at /games I should move it to /originals?
It's that correct?

Reply:
correct; please refer to the schema and readme

Reply:
done, please let me know! thanks